api/go.mod

```diff
@@ -1,19 +1,19 @@
 module kubevirt.io/ssp-operator/api
 
-go 1.19
+go 1.20
 
 require (
-	github.com/openshift/api v0.0.0-20230228142948-d170fcdc0fa6 // release-4.13
-	k8s.io/apimachinery v0.26.10
-	kubevirt.io/containerized-data-importer-api v1.55.2
+	github.com/openshift/api v0.0.0-20231130063041-be20875a1373 // release-4.15
+	k8s.io/apimachinery v0.28.3
```

Comment:
There is `v0.28.4` already, can we use it?

Reply:
done